src/cli/init_command.zig

```diff
@@ -337,7 +337,14 @@ pub const InitCommand = struct {
         }
 
         if (fields.entry_point.len > 0 and !exists(fields.entry_point)) {
-            var entry = try std.fs.cwd().createFile(fields.entry_point, .{ .truncate = true });
+            const cwd = std.fs.cwd();
+            if (std.fs.path.dirname(fields.entry_point)) |dirname| {
+                if (!(dirname.len == 0 and dirname[0] == '.')) {
```

Comment:
I think you meant 

```suggestion
                if (strings.eqlComptime(dirname, “.”) and strings.eqlComptime(dirname, “./)) {
```

dirname will return null if it has no separator so checking for .len == 0 will never be reached

Reply:
ah, a typo.

Reply:
Fixed. `dirname` will not return `"./"`, comparing to `"."` is enough here.